project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala

```diff
@@ -69,14 +71,6 @@ object Docker {
     }
   }
 
-  /** Other customizations for sbt-docker. */
-  lazy val buildOptions: Initialize[BuildOptions] = Def.setting {
-    BuildOptions(
-      cache = false,
```

Comment:
As far as I can tell, it should be safe to cache when building these images. Docker uses file hashes for invalidating `ADD` layers so we shouldn't be at risk of loading a stale assembly jar.